src/app/components/Metadata/utils/index.tsx

```diff
@@ -1,16 +1,26 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 import React from 'react';
+import { Services } from '#app/models/types/global';
+import { AppleItunesApp, AlternateLink, IconSizes, IconType } from '../types';
 
-export const getIconAssetUrl = (service, size) =>
+export const getIconAssetUrl = (service: Services, size: string) =>
   `https://static.files.bbci.co.uk/ws/simorgh-assets/public/${service}/images/icons/icon-${size}.png`;
 
-const createIconLinks = (service, iconSizes, iconType) => {
-  return iconSizes.map(size => {
+const createIconLinks = (
+  service: Services,
+  iconSizes: IconSizes,
+  iconType: string,
+) => {
+  const iconSizesForType = iconSizes[iconType as IconType];
+
+  return iconSizesForType.map(size => {
     const iconAssetUrl = getIconAssetUrl(service, size);
     const key = `${service}-${size}`;
 
     if (iconType === 'icon') {
       return (
         <link
+          /* @ts-ignore:   Property 'key' does not exist on type 'LinkProps & { css?: Interpolation<Theme>; }'.ts(2322) */
           key={key}
```

Comment:
I tried removing the `key` props from here and below and it seems ok 🤷 React mustn't need a unique identifier for some elements.

Reply:
I keep getting TS errors when I remove the `ts-ignore`, and failing unit tests when I remove the key prop 🤯 

```
 FAIL   Unit Tests  src/app/components/Metadata/index.test.tsx (8.307 s)
  ● should render the dir and lang attribute

    Test failed because a React warning was detected. Please fix the following:
    Error: Test failed because a React warning was detected. Please fix the following:
    Warning: Each child in a list should have a unique "key" prop.%s%s See https://reactjs.org/link/warning-keys for more information.%s
```

So I am going to leave this as is for now!